src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,21 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User;
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <>
+      <a className="UserInfo" href={`mailto:${email}`}>
+        {name}
+      </a>
+    </>
```

Comment:
`Don't use React fragment, if you already have a wrapper`

```suggestion
    <a className="UserInfo" href={`mailto:${email}`}>
      {name}
     </a>
```